ZenPacks/zenoss/Dashboard/facades.py

```diff
@@ -185,7 +208,17 @@ def getTopLevelOrganizers(self, uid):
     def getMultiGraphReports(self, uid='/zport/dmd/Reports'):
         results = []
         obj = self._getObject(uid)
-        searchresults = ICatalogTool(obj).search(MultiGraphReport)
+
+        try:
```

Comment:
this try/except is repeated in multiple places. maybe add a get_catalog_method? or a methods that returns a boolean indicating if model catalog is available. IModelCatalogTool.search and ICatalogTool.search should be compatible.